app/src/main/java/org/oppia/app/databinding/DrawableBindingAdapters.java

```diff
@@ -0,0 +1,57 @@
+package org.oppia.app.databinding;
+
+import android.graphics.drawable.GradientDrawable;
+import android.view.View;
+import android.widget.Button;
+
+import androidx.annotation.ColorInt;
+import androidx.annotation.DrawableRes;
+import androidx.annotation.NonNull;
+import androidx.databinding.BindingAdapter;
+
+import org.oppia.app.R;
+
+/**
+ * Holds all the custom binding adapters that set background drawables.
```

Comment:
Optional nit here & elsewhere: if a Javadoc can fit within one line, you can shorten it, e.g.:

```java
/** Holds all the custom binding adapters that set background drawables. */
```

Reply:
Done!